helpers/sizes.ts

```diff
@@ -0,0 +1,24 @@
+export function convertToPixels(
+  size: string,
+  context = document.documentElement,
+) {
+  // Create a temporary element
+  const tempEl = document.createElement('div')
```

Comment:
This is expensive. You could reuse a hidden element here if you wanted to optimize more for performance. You could also cache the result in a map by size.